src/components/molecules/Visualizer/WidgetAlignSystem/hooks.ts

```diff
@@ -0,0 +1,125 @@
+import { useCallback, useEffect, useState } from "react";
+
+import { Widget as WidgetType } from "../Widget";
+
+export type Alignments = "start" | "centered" | "end";
+
+export type Location = {
+  zone?: string;
+  section?: string;
+  area?: string;
```

Comment:
Stringer type is recommended 
```suggestion
  area?: "top" | "middle" | "bottom";
```